### DISCUSSION TITLE
Horizontal Scroll bar on Appshell

### DISCUSSION THREAD

**Discussion by lokeshwarobuli** [mantinedev/mantine#2619] [Q&A] [answered]
When I include Appshell in my app, it creates an horizontal scroll bar at the bottom of the screen, eventhough the contents are not overflowing. When I remove the appshell and wrap the contents inside a regular div, the horizontal scrollbar disappers.

Any tips/help/workarounds please

Thanks

**Comment by rtivital** [comment, marked as answer]
Try setting withNormalizeCSS and withGlobalStyles on MantineProvider

**Reply by lokeshwarobuli** [reply]
Is it absolutely necessary to use the Mantine Provider, because right now I haven't set the Mantine Provider in my app. Should I set it up?

Thanks

**Reply by rtivital** [reply]
Yes, you should set it up
